src/Symfony/Bundle/FrameworkBundle/Command/ConfigDebugCommand.php

```diff
@@ -101,20 +101,22 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
             $io->writeln(Yaml::dump([$extensionAlias => $config], 10));
 
-            return;
+            return 0;
         }
 
         try {
             $config = $this->getConfigForPath($config, $path, $extensionAlias);
         } catch (LogicException $e) {
             $errorIo->error($e->getMessage());
 
-            return;
+            return 1;
```

Comment:
that one should be done in older branches too, as it fixes a bug (as a success exit code was used)

Reply:
The lowest branch this needs to be fixed in is 3.4, right?

Reply:
That's correct.